src/main/java/calculator/AddCalculator.java

```diff
@@ -0,0 +1,9 @@
+package calculator;
+
+import java.util.Arrays;
+
+public class AddCalculator {
+    public static int calculate(String[] numbers){
+        return Arrays.stream(numbers).mapToInt(number -> Integer.parseInt(number)).sum();
```

Comment:
```suggestion
        return Arrays.stream(numbers)
                            .mapToInt(number -> Integer.parseInt(number))
                            .sum();
```

혹시 parseInt 가 실패하면 어떻게되나요?